### PR TITLE
Fix checking for Android SDK

### PR DESCRIPTION
**Pull request by teobugslayer**
Fix error handling on process spawn

Fixes http://teampulse.telerik.com/view#item/279833

### CODE REVIEW

File: child-process.ts [hunk 1]
```diff
@@ -79,11 +79,17 @@ export class ChildProcess implements IChildProcess {
...
 						errorMessage += util.format(' Error output: \n %s', capturedErr);
 					}
-					future.throw(errorMessage);
+					future.throw(new Error(errorMessage));
 				}
 			}
 		});
 
+		if (event !== "error") {
+			childProcess.on("error", (err: Error) => {
```
Comment (Fatme) [inline child-process.ts]: maybe `childProcess.once("error")`

File: child-process.ts [hunk 2]
```diff
@@ -79,11 +79,17 @@ export class ChildProcess implements IChildProcess {
 					if (capturedErr) {
 						errorMessage += util.format(' Error output: \n %s', capturedErr);
 					}
-					future.throw(errorMessage);
+					future.throw(new Error(errorMessage));
```
Comment (Fatme) [inline child-process.ts]: Why we need `new Error`?

File: child-process.ts (hunk 1 above)
Comment (teobugslayer) [inline child-process.ts]: Sure!

File: child-process.ts (hunk 2 above)
Comment (teobugslayer) [inline child-process.ts]: Because otherwise we throw string, which is allowed by the language but not a good practice.

**Comment by Fatme** [comment]
:+1:

**Pull request merged**